src/client/Microsoft.Identity.Client/Extensibility/AbstractConfidentialClientAcquireTokenParameterBuilderExtension.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Extensibility
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class AbstractConfidentialClientAcquireTokenParameterBuilderExtension
```

Comment:
Also would this apply to all confidential client flows or only client credentials?

Reply:
1. I'm thinking of keeping it in a separate namespace as this is really quite advanced.
2. All confidential clients, since they all need an assertion (even if we build one for them via the cert)

Reply:
As discussed, I think it's ok to have it in extensibility namespace (especially if it's only to support legacy behavior).